G/GAP_lib/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase, Pkg
+
+name = "GAP"
+version = v"4.11.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/gap-system/gap.git", "069a6497424113dc5e5ffe133be9da5bfe6acb24"),
+#    ArchiveSource("https://github.com/gap-system/gap/releases/download/v$(version)/gap-$(version)-core.tar.bz2",
+#                  "6637f66409bc91af21eaa38368153270b71b13b55b75cc1550ed867c629901d1"),
+    ArchiveSource("https://github.com/gap-system/gap/releases/download/v$(version)/packages-required-v$(version).tar.gz",
+                  "29ab6e2752f39d22e3f0a19e5bcbfec1710993b4cfd52337a0ca10fda6a76537";
+                  unpack_target="pkg"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/gap*
+
+mv ../pkg .
+find pkg -name '._*' -exec rm \{\} \; # unwanted files
+
+# run autogen.sh if compiling from it source and/or if configure was patched
+./autogen.sh
+
+# compile a native version of GAP so we can use it to generate the manual
+# (the manual is only in FULL gap release tarballs, not in the -core tarball
+# nor in git snapshots)
+mkdir native-build
+cd native-build
+#apk add gmp-dev zlib-dev
+../configure --build=${MACHTYPE} --host=${MACHTYPE} CC=${CC_BUILD} CXX=${CXX_BUILD} --with-gmp=/usr
```

Comment:
```suggestion
../configure --build=${MACHTYPE} --host=${MACHTYPE} CC=${CC_BUILD} CXX=${CXX_BUILD} --with-gmp=${prefix}
```
?

Reply:
yeah, that came from an older version of the code where I used `apk install gmp`.